adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -101,6 +104,7 @@ - (void)initWithAppId:(nonnull NSString *)appId
   if (delegate) {
     NSString *priorityPlacementID = delegate.desiredPlacement;
     [initOptions setObject:priorityPlacementID forKey:VungleSDKInitOptionKeyPriorityPlacementID];
+    _prioritizedPlacementID = [priorityPlacementID copy];
```

Comment:
I wonder if we need to create and add "delegate" at this point before calling init (which leads to requesting ad)?
Just curious.
I run the project and verified it's working without problem, so I approved this for now for your to build timely.
If you need to change it, I will review again. Please let me know.

Reply:
@ashinagawa Hi Aki, thanks a lot for the reminder, in the top of this method `initWithAppId:delegate:`, we already added the "delegate":

 if (delegate) {
    GADMAdapterVungleMapTableSetObjectForKey(_initializingDelegates, delegate.desiredPlacement,
                                             delegate);
  }

^_^